apps/classroomio-com/src/routes/bootcamps/+page.svelte

```diff
@@ -0,0 +1,257 @@
+          </p>
+        </div>
+      </div>
+      <div class="w-full md:w-[100%] p-6 border border-[#D9E0F5] rounded-2xl flex flex-col gap-y-6">
+        <div>
+          <img src="/employee-training/pie-chart.svg" alt="pie-chart" />
+        </div>
+        <div class="w-full flex flex-col justify-start align-start gap-y-2">
+          <h1 class="w-full text-xl font-medium font-[#282828]">Data-Driven Insights</h1>
+          <p class=" w-full text-base font-[#656565] font-light">
+            Gain valuable insights into employee performance and learning patterns.
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section
+    class="w-full bg-[#F1F6FF] flex flex-col items-start justify-start gap-y-16 lg:py-24 py-16 lg:px-28 px-8"
+  >
+    <div
+      class="w-full gap-y-4 flex flex-col lg:items-center items-start lg:justify-center justify-start"
+    >
+      <div class="w-fit py-2 px-4 rounded-full border-[#C2D2FF] border-2 text-center">
+        <p class=" text-sm md:text-base text-[#4B5563] text-center font-medium">
+          Explore our features
+        </p>
+      </div>
+      <div
+        class="lg:w-full flex flex-col lg:items-center items-start lg:justify-center justify-start gap-y-4"
+      >
+        <h1 class="md:text-5xl text-3xl font-medium text-[#282828]">
+          What makes us the right choice
+        </h1>
+        <p class="text-[#656565] text-xl font-light lg:text-center lg:w-[80%]">
+          Discover why we’re the perfect partner for your employee training needs. Experience
+          innovative solutions designed to reduce stress and enhance productivity.
+        </p>
+      </div>
+    </div>
+
+    <div class="w-full flex flex-col justify-start items-start lg:gap-y-24 gap-y-16">
+      <div
+        class="w-full flex lg:flex-row flex-col lg:justify-between justify-start lg:items-center items-start gap-y-8"
+      >
+        <div class="w-full lg:w-[40%] gap-y-4 flex flex-col items-start justify-start">
+          <img src="/employee-training/team-mgnt.svg" alt="team icon" />
+          <div class="md:w-full flex flex-col items-start justify-start gap-y-4">
+            <h1 class="md:text-5xl text-3xl font-medium text-[#282828]">Team management</h1>
+            <p class="text-[#656565] text-xl font-light">
+              Company administrators can invite their team members to collaborate on course creation
+              within classroomio
+            </p>
+          </div>
+          <button
+            class="py-4 px-8 border border-[#1D4ED8] outline-transparent rounded text-[#1D4ED8] text-sm font-semibold"
+            >Book a Demo</button
+          >
+        </div>
+
+        <div>
+          <img src="/employee-training/team-mgnt2.svg" alt="team management feature ui" />
+        </div>
+      </div>
+      <div
+        class="w-full flex lg:flex-row-reverse flex-col lg:justify-between justify-start lg:items-center items-start gap-y-8"
```

Comment:
can you please record a demo for this page? the `lg:` attribute caters for large screens and i don't see you catering for medium screens here so this might be buggy for medium devices. the scope i use is to set the class normally then use the `md:` attributes since that caters for medium devices upwards